client/my-sites/checklist/main.jsx

```diff
@@ -49,6 +49,11 @@ class ChecklistMain extends PureComponent {
 	 */
 	maybeRedirectJetpack( prevProps = {} ) {
 		if (
+			/**
+			 * Only send Jetpack send users to plans if a checklist will be presented. Otherwise,
```

Comment:
I think there's a _send_ too much, isn't it?

Reply:
😮 
✅ b218638